internal/app/feed/feed_action.go

```diff
@@ -28,6 +28,7 @@ type Action struct {
 	AuditTitle      string    `json:"audit_title" csv:"audit_title"`
 	AuditItemID     string    `json:"audit_item_id" csv:"audit_item_id"`
 	AuditItemLabel  string    `json:"audit_item_label" csv:"audit_item_label"`
+	OrganisationID  string    `json:"organisation_id" csv:"organisation_id" gorm:"index:idx_act_organisation_id"`
```

Comment:
we don't want a new index. This should be added to `idx_act_modified_at` index and we have to make sure that the query that is generated uses the index. This should be done through some profiling.